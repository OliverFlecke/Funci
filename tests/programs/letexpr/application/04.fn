-- more partial app
main :: Int =
 let f x :: (Int -> Int) = x + 1
 in let g :: (Int -> Int) = f
    in g 1
