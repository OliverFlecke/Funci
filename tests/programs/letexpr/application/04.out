Number (I 2) (Unit [])